Applications/Android/NNDetector/app/src/main/jni/image.h

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * @file   image.h
+ * @date   24 Oct 2023
+ * @brief  image decoder
+ * @author Umberto Michieli(u.michieli@samsung.com)
```

Comment:
Could you check the copy right once again? 
https://github.com/nnstreamer/nntrainer/blob/main/Applications/Android/ResnetJNI/app/src/main/jni/image.h

Reply:
> Could you check the copy right once again? https://github.com/nnstreamer/nntrainer/blob/main/Applications/Android/ResnetJNI/app/src/main/jni/image.h

done

Reply:
done